testsuite/mlir/mlir-parse/dialect.mlir

```diff
@@ -23,38 +23,40 @@
 // }
 // ```
 
+!C = type !verona.class<"C">
+!D = type !verona.class<"D", "f" : meet<U64, imm>, "g" : meet<!C, mut>>
 module {
   verona.class @C {
   }
 
   verona.class @D {
     verona.field "f" : !verona.meet<U64, imm>
-    verona.field "g" : !verona.meet<class<"C">, mut>
+    verona.field "g" : !verona.meet<!C, mut>
```

Comment:
From the lack of changes in the output, I imagine this is identical to the previous syntax, which reinforces my point above.